libraries/AP_HAL_ChibiOS/hwdef/NucleoH743-IKS02/hwdef.dat

```diff
@@ -0,0 +1,129 @@
+# hw definition file for processing by chibios_hwdef.py
```

Comment:
This should be includiong a base `hwdef.dat` from somewhere and only updating/overriding what it needs to.